notebooks/120-tf2-object-detection-to-openvino/120-tf2-object-detection-to-openvino.ipynb

```diff
@@ -0,0 +1,578 @@
+{
```

Comment:
<p>probably we should add some words about TF object detection API (as this library is source of your testing model)</p><p> </p><p>As tensorflow1.x deprecated 2 years ago if I right remember, can we remove suffix 2 from name of notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
and everywhere in notebook

Reply:
TF suffix fixed, TF OD API description added